src/main/java/com/zombiecastlerush/util/Prompter.java

```diff
@@ -18,12 +18,59 @@ static String getUserInput(String displayMessage , String... args) {
         return result;
     }
 
-    static void movePlayer(Player player) {
-        List<Room> availableRooms = player.getCurrentPosition().getConnectedRooms();
-        String moveString = Prompter.getUserInput("\nYou are in " + player.getCurrentPosition() + "\n\nTo go somewhere, please type go and one of the following available locations: " + availableRooms);
-        List<String> moveList = Parser.parse(moveString);
-        if (moveList != null) {
-            player.moveTo(moveList.get(1));
+    static void controller(Player player) {
```

Comment:
It would probably better to make this a verb. Maybe just "advanceGame" or "advancePlayer".